src/requisitionMerge.js

```diff
@@ -163,19 +163,23 @@ function requisitionItemsMerge(incomingRequisitionLines, outgoingRequisitionLine
  * @return {Object} the updated eSIGL requisition with incoming values applied.
  */
 export default function requisitionMerge(incomingRequisition, outgoingRequisition) {
-  const { requisitionLines: incomingLines } = incomingRequisition;
+  const { requisitionLines: incomingLines, custom_data } = incomingRequisition;
   const { fullSupplyLineItems: outgoingLines, regimenLineItems } = outgoingRequisition;
+  const regimenData = custom_data.regimenData || [];
```

Comment:
I can't quite remember if custom_data is always at least an empty object? If not, and it is undefined, this could throw an error?

`(custom_data && custom_data.regimenData) || [];`

Reply:
yes, good point. Should be populated to get this far ( has to be a program based requisition etc ). can't trust anything though..

Reply:
I think if the requisition doesn't have regimen data (which some programs don't seem to), then custom_data may be null